### ISSUE TITLE
Active Column Highlighting Setting

### ISSUE THREAD

**Issue by JustSomeGuy805** [SeeSharpSoft/intellij-csv-validator#371]
Is there a way remove this Active Column Highlighting setting? Whenever I do a "find" it will highlight the active column plus the matches. I just want to see the highlighted matches from "find".

**Comment by SeeSharpSoft** [comment]
No, there is no such setting. The active column highlighting is meant to be only triggered by manually marking text. I noticed that the marking of search results also triggers the column highlighting, but it can be easily dismissed by just clicking anywhere in the text editor. The search results will still remain highlighted.

However, I will dig into the unwanted column highlighting while searching in the first place. Thanks for reporting.
